app/models/spree/user_decorator.rb

```diff
@@ -18,10 +18,16 @@
   accepts_nested_attributes_for :ship_address
 
   attr_accessible :enterprise_ids, :enterprise_roles_attributes, :enterprise_limit, :bill_address_attributes, :ship_address_attributes
-  after_create :send_signup_confirmation
 
   validate :limit_owned_enterprises
 
+  # We use the same options as Spree and add :confirmable
+  devise :confirmable, reconfirmable: true
+  handle_asynchronously :send_confirmation_instructions
+  handle_asynchronously :send_on_create_confirmation_instructions
+  # TODO: Later versions of devise have a dedicated after_confirmation callback, so use that
```

Comment:
Any link to the code of future Devise? Just add it here in the TODO comment please, thanks!